sqlDocker/finwire_parsing.py

```diff
@@ -0,0 +1,111 @@
+import pandas as pd
+import re
+import os
+
+# Helper functions
+
+# Get a list of column ranges according to the type of schema
+def get_col_spec_list( schema_df ):

```

Comment:
```suggestion
def get_col_spec_list( schema_df ):
   '''Get a list of column ranges according to the type of schema'''
```
We should be using docstring